common/src/main/java/io/netty/util/HashedWheelTimer.java

```diff
@@ -271,8 +271,8 @@ public HashedWheelTimer(
 
         if (duration < MILLISECOND_NANOS) {
             if (logger.isWarnEnabled()) {
-                logger.warn("Configured tickDuration %d smaller then %d, using 1ms.",
-                            tickDuration, MILLISECOND_NANOS);
+                logger.warn(String.format("Configured tickDuration %d smaller then %d, using 1ms.",
```

Comment:
@szhnet wouldn't it be better to just replace the `%d`s with `{}`s?

Reply:
Right, {} is better.